doc/arkode/guide/source/Butcher.rst

```diff
@@ -477,6 +513,7 @@ or :c:func:`ARKodeButcherTable_LoadERK`.
 Accessible via the string ``"ARKODE_SOFRONIOU_SPALETTA_5_3_4"`` to
 :c:func:`ARKStepSetTableName`, :c:func:`ERKStepSetTableName` or
 :c:func:`ARKodeButcherTable_LoadERKByName`.
+This is the default 4th order explicit method
```

Comment:
I'm not sure why periods were added in-between the text and all previous references for where the methods came from, but to be consistent a period should either be added here (or removed above).